src/cocol/node/ledger/api.cr

```diff
@@ -49,6 +51,11 @@ post "/blocks/pow" do |env|
       response: "Bad Request"
     )
   end
+  halt(
+    env,
+    status_code: 422,
```

Comment:
are you defining your own status codes here?

Reply:
nope: https://httpstatuses.com/422